### PR TITLE
lc113

### PR DESCRIPTION
**Pull request by victorFFFF**

### CODE REVIEW

File: victorLi/assignments/treesDFS/lc113/lc113.js [hunk 1]
```diff
@@ -12,6 +12,21 @@
 //  / |   | \
 // 4  5   2  7
 
+/*traverse using DFS
+  if path = target then push stack into allPaths array
```
Comment (knrodriguez) [inline victorLi/assignments/treesDFS/lc113/lc113.js:16]: When do you push stack into allPaths? More detail on this approach would help make it clearer how you code is attempting to satisfy the prompt.

File: victorLi/assignments/treesDFS/lc113/lc113.js [hunk 2]
```diff
@@ -12,6 +12,21 @@
...
+            12
+          7    1
+        4    10   5
+                 5  3
+
+          path 1 : 12 7 4
+          path 2 : 12 1 10
+          path 3 : 12 1 5 5     
+*/
+
```
Comment (knrodriguez) [inline victorLi/assignments/treesDFS/lc113/lc113.js:29]: Missing edge cases.

File: victorLi/assignments/treesDFS/lc113/lc113.js [hunk 3]
```diff
@@ -22,18 +37,67 @@ class TreeNode {
...
+    return allPaths;
+};
 
+let isLeaf = (node) =>{
+  if( !node.right && !node.left )
+    return true
+  else
+    return false
+}
```
Comment (knrodriguez) [inline victorLi/assignments/treesDFS/lc113/lc113.js:92]: Don't forget to write it twice to build up your muscle memory.

File: victorLi/assignments/treesDFS/lc113/lc113.js [hunk 4]
```diff
@@ -22,18 +37,67 @@ class TreeNode {
...
 
 var root = new TreeNode(12)
 root.left = new TreeNode(7)
 root.right = new TreeNode(1)
 root.left.left = new TreeNode(4)
 root.right.left = new TreeNode(10)
 root.right.right = new TreeNode(5)
+root.right.right.left = new TreeNode(5)
+root.right.right.right = new TreeNode(3)
```
Comment (knrodriguez) [inline victorLi/assignments/treesDFS/lc113/lc113.js:101]: Nice job adding a custom test case, but make sure one of the test cases you make checks the edge case(s) you list beforehand.

**Review by knrodriguez — APPROVED** [review]
